package.json

```diff
@@ -2,9 +2,9 @@
   "name": "root",
   "version": "0.0.0",
   "private": true,
-  "packageManager": "pnpm@10.10.0",
+  "packageManager": "pnpm@10.11.0",
```

Comment:
This will conflict with #3622, so who maybe let renovate do the updating and close the other PR?

Reply:
yes